include/libtorrent/storage.hpp

```diff
@@ -247,9 +243,9 @@ namespace libtorrent
 		// error. If there's an error, the ``storage_error`` must be filled out
 		// to represent the error that occurred.
 		virtual int readv(span<file::iovec_t const> bufs
-			, int piece, int offset, int flags, storage_error& ec) = 0;
+			, int piece, int offset, int flags, int open_flags, storage_error& ec) = 0;
```

Comment:
the thing that worries me a bit here is how all the ``int`` arguments aren't typed. It makes it easy to get the order wrong. obviously this is not a new problem, it's just compounding.